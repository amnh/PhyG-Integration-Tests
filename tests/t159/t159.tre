(Afrikaans:6054.2,(PharasaGreek:4526.35,(Luxemburgish:3061.15,(KolschColognian:4319.0,(GreekModern:6166.549999999999,(ArmenianEastern:3960.55,(AncientGreek:5542.450000000001,(AlbanianTosk:3790.55,(((EnglishBritish:2784.0,(Luwian:4108.5,Yiddish:4229.65)HTU67:2420.85)HTU68:2559.65,((Dutch:1848.15,OldPrussian:1949.35)HTU75:2275.65,((EasternFrisianSaterland:1880.55,Scots:2086.05)HTU71:1772.85,((German:2614.15,(Latgalian:2423.2,Lithuanian:2877.1000000000004)HTU97:1103.05)HTU73:1522.65,(SchwyzerdutschSwissGerman:5538.450000000001,(CentralFranconianRipuarian:3307.15,SouthernTsakonian:4532.55)HTU98:1818.35)HTU74:3306.6)HTU72:1611.15)HTU96:1778.2)HTU76:2010.35)HTU94:2956.15,((WesternFrisian:4296.25,(Latvian:3873.85,(Breton:3152.55,(((Cornish:2349.15,Irish:2113.55)HTU78:1072.45,(AncientIonicGreek:5144.1,ClassicalArmenian:2737.15)HTU80:1618.15)HTU81:1635.35,(Assamese:4324.85,(Faroese:2725.85,Icelandic:6086.85)HTU79:2197.15)HTU82:2690.05)HTU83:1591.55)HTU84:1808.35)HTU85:2203.15)HTU77:4973.7,(((Manx:2576.15,(Samogitian:3441.95,(OldIrish:1534.55,ScottishGaelic:1448.05)HTU51:1188.8000000000002)HTU52:1025.15)HTU53:7288.950000000001,(ArmenianWestern:9299.6,(PennsylvaniaDutch:9521.55,((LowSaxon:3045.35,(NorthFrisianOomrang:7772.55,OldHighGerman:2996.0)HTU54:2043.15)HTU58:10619.3,((SanskritVedic:11738.25,(EnglishAmerican:11281.65,(Norweigan:12932.449999999999,(Gothic:3997.65,Swedish:3186.75)HTU55:10338.15)HTU56:7719.75)HTU59:6936.65)HTU60:6182.15,(Avestan:3680.25,Danish:3836.65)HTU61:9184.55)HTU62:6136.35)HTU63:7124.85)HTU64:5269.45)HTU65:4924.15)HTU57:4977.35,(TocharianB:3195.7,(TocharianA:2127.65,Welsh:2334.85)HTU69:1480.15)HTU70:6267.15)HTU93:4322.05)HTU95:4416.15)HTU66:3515.25)HTU86:2122.15)HTU87:2299.15)HTU88:3070.75)HTU89:3480.4500000000003)HTU90:2688.75)HTU91:3251.75)HTU92:4083.2)HTU50[157437.0];
(Afrikaans:6208.7,(PharasaGreek:4525.35,(Luxemburgish:3055.65,(KolschColognian:4313.5,(GreekModern:6200.549999999999,(ArmenianEastern:4043.05,(AncientGreek:5479.950000000001,(AlbanianTosk:3932.55,((((Luwian:3806.5,(SanskritVedic:3882.35,Yiddish:4036.65)HTU64:1851.65)HTU65:2484.85,(EnglishBritish:3629.1,Gothic:4669.25)HTU66:2090.5499999999997)HTU68:8692.75,((Dutch:1767.25,OldPrussian:1947.25)HTU76:8889.65,((EasternFrisianSaterland:1854.05,Scots:2104.55)HTU73:9072.75,((SchwyzerdutschSwissGerman:13848.35,(CentralFranconianRipuarian:3282.65,SouthernTsakonian:4555.05)HTU98:10049.95)HTU75:7350.3,(Latgalian:2386.2,Lithuanian:3005.6000000000004)HTU97:9430.75)HTU74:6149.15)HTU96:5531.4)HTU77:5331.15)HTU94:5666.65,((WesternFrisian:4471.05,(Breton:3137.95,(((Cornish:2394.15,Irish:2091.05)HTU79:1124.45,(AncientIonicGreek:5146.6,ClassicalArmenian:2792.15)HTU81:1637.15)HTU82:1684.85,(Assamese:4359.85,(Faroese:2729.35,Icelandic:6094.35)HTU80:2171.15)HTU83:2723.55)HTU84:1736.15)HTU85:1842.65)HTU78:5134.55,(((OldIrish:1598.05,ScottishGaelic:1512.05)HTU51:2986.15,(ArmenianWestern:4098.15,(PennsylvaniaDutch:4083.55,((German:3383.65,(LowSaxon:3618.35,(NorthFrisianOomrang:7228.55,OldHighGerman:3616.0)HTU52:2768.65)HTU55:4010.85)HTU57:2392.6,((EnglishAmerican:2499.75,(Norweigan:3750.4500000000003,Swedish:2872.75)HTU53:3231.15)HTU56:1680.65,(Avestan:4213.65,Danish:3165.25)HTU58:1802.55)HTU59:2243.75)HTU60:1993.35)HTU61:2372.4)HTU62:2321.65)HTU54:3041.5499999999997,((Samogitian:2982.35,TocharianB:3290.8)HTU69:1600.65,(Latvian:2303.4,(Welsh:1735.4499999999998,(Manx:1919.25,TocharianA:2622.05)HTU67:1040.65)HTU70:1018.4499999999999)HTU71:1348.15)HTU72:3633.6)HTU93:4534.55)HTU95:4868.8)HTU63:7238.85)HTU86:2300.65)HTU87:2154.15)HTU88:2985.75)HTU89:3520.4500000000003)HTU90:2699.25)HTU91:3258.25)HTU92:4479.7)HTU50[157119.4];
(Afrikaans:5430.2,(PharasaGreek:4543.35,(Luxemburgish:3071.65,(KolschColognian:4331.5,(GreekModern:6214.549999999999,(ArmenianEastern:4008.55,(AncientGreek:5556.950000000001,(AlbanianTosk:3876.55,((((SanskritVedic:3958.15,Yiddish:3915.35)HTU65:1721.65,(EnglishBritish:3540.6,Gothic:4803.75)HTU66:1515.55)HTU67:8522.65,((Dutch:1758.75,OldPrussian:1965.25)HTU73:8739.15,(EasternFrisianSaterland:9761.65,((SchwyzerdutschSwissGerman:13101.85,(SouthernTsakonian:11956.05,(CentralFranconianRipuarian:3622.65,Luwian:4039.1)HTU75:10292.65)HTU98:6722.45)HTU72:7601.3,(Latgalian:2302.2,Lithuanian:3062.1000000000004)HTU97:9008.75)HTU71:5927.65)HTU96:5450.9)HTU74:5423.75)HTU94:5450.15,((WesternFrisian:4713.75,(Latvian:3959.35,(Breton:3389.05,(((Cornish:2427.15,Irish:2002.55)HTU77:1010.4499999999999,(AncientIonicGreek:5087.6,ClassicalArmenian:2798.15)HTU79:1683.15)HTU81:1872.85,((Faroese:2017.85,Icelandic:6628.35)HTU78:1503.15,(Assamese:5081.85,Scots:2359.35)HTU80:1285.65)HTU82:2279.0499999999997)HTU83:1591.55)HTU84:1881.85)HTU85:2080.65)HTU76:5330.2,(((Manx:2176.75,(OldIrish:1576.05,ScottishGaelic:1420.05)HTU51:928.2)HTU52:2995.9500000000003,(ArmenianWestern:4010.2,(PennsylvaniaDutch:4098.55,((German:3401.65,(LowSaxon:3593.35,(NorthFrisianOomrang:7266.05,OldHighGerman:3590.0)HTU53:2745.65)HTU56:3999.85)HTU58:2428.6,((EnglishAmerican:2531.25,(Norweigan:3744.4500000000003,Swedish:2865.75)HTU54:3179.15)HTU57:1708.65,(Avestan:4242.15,Danish:3122.75)HTU59:1824.05)HTU60:2254.25)HTU61:2021.35)HTU62:2396.35)HTU63:2366.65)HTU55:3008.15,((Samogitian:3083.85,TocharianB:3217.3)HTU68:1386.65,(TocharianA:2248.95,Welsh:2228.05)HTU69:1400.15)HTU70:3732.65)HTU93:4728.65)HTU95:5064.65)HTU64:7529.75)HTU86:2257.65)HTU87:2174.15)HTU88:2995.75)HTU89:3527.4500000000003)HTU90:2700.25)HTU91:3257.25)HTU92:3990.7)HTU50[156868.1];
(Afrikaans:5014.2,(PharasaGreek:4545.35,(Luxemburgish:3062.65,(KolschColognian:4326.0,(GreekModern:6198.049999999999,(ArmenianEastern:4001.05,(AncientGreek:5495.450000000001,(AlbanianTosk:3802.55,(((EnglishBritish:3154.1,(Luwian:3792.0,(SanskritVedic:3613.35,Yiddish:4341.65)HTU65:1730.15)HTU66:2534.25)HTU67:9085.65,(Welsh:10334.95,((Dutch:1763.55,OldPrussian:1939.4499999999998)HTU74:9262.15,(EasternFrisianSaterland:10626.85,((German:2189.15,(Latgalian:2548.7,Lithuanian:2900.6000000000004)HTU97:1500.05)HTU72:9911.15,(SchwyzerdutschSwissGerman:13796.449999999999,(CentralFranconianRipuarian:3348.65,SouthernTsakonian:4368.55)HTU98:11133.35)HTU73:8042.1)HTU71:6068.55)HTU96:5923.2)HTU75:5489.25)HTU70:5798.25)HTU94:5522.45,((WesternFrisian:4602.75,(Latvian:3923.85,(Breton:3387.55,(((Cornish:2393.65,Irish:1984.55)HTU77:1011.9499999999999,(AncientIonicGreek:5139.1,ClassicalArmenian:2741.15)HTU79:1633.65)HTU81:1880.35,((Faroese:2007.35,Icelandic:6629.85)HTU78:1510.65,(Assamese:5068.85,Scots:2342.85)HTU80:1283.65)HTU82:2284.5499999999997)HTU83:1599.55)HTU84:1943.35)HTU85:2142.15)HTU76:6153.7,(((Manx:2046.15,(OldIrish:1579.55,ScottishGaelic:1420.55)HTU51:1058.8000000000002)HTU52:3704.9500000000003,(ArmenianWestern:4698.1,(PennsylvaniaDutch:4648.55,((LowSaxon:3199.85,(NorthFrisianOomrang:7588.55,OldHighGerman:3238.5)HTU53:2310.65)HTU57:5288.299999999999,((EnglishAmerican:3576.75,(Norweigan:4239.950000000001,(Gothic:3881.15,Swedish:3033.75)HTU54:1912.65)HTU55:3002.65)HTU58:2240.15,(Avestan:4245.15,Danish:3201.25)HTU59:2802.55)HTU60:2869.75)HTU61:2513.85)HTU62:2843.4500000000003)HTU63:2793.15)HTU56:3378.85,(TocharianA:2621.65,(Samogitian:3215.95,TocharianB:3083.7)HTU68:1395.65)HTU69:4609.65)HTU93:5139.55)HTU95:5402.75)HTU64:7971.15)HTU86:2315.15)HTU87:2235.15)HTU88:3007.75)HTU89:3526.9500000000003)HTU90:2698.25)HTU91:3252.25)HTU92:3732.2)HTU50[156528.2];
(Afrikaans:5088.7,(PharasaGreek:4573.35,(Luxemburgish:2879.65,(KolschColognian:4467.5,(GreekModern:6224.549999999999,(ArmenianEastern:3951.05,(AncientGreek:5584.450000000001,(AlbanianTosk:3835.05,(((EnglishBritish:2682.0,(SanskritVedic:3814.15,Yiddish:4102.35)HTU66:2070.65)HTU67:8672.65,(Welsh:9681.35,((Dutch:1799.15,OldPrussian:1976.35)HTU74:8857.65,((EasternFrisianSaterland:1940.55,Scots:2107.05)HTU71:9146.15,((SchwyzerdutschSwissGerman:13589.35,(SouthernTsakonian:12534.05,(CentralFranconianRipuarian:3657.65,Luwian:4053.1)HTU76:10851.65)HTU98:7068.95)HTU73:8000.8,(Latgalian:2365.2,Lithuanian:2974.6000000000004)HTU97:9636.25)HTU72:6244.75)HTU96:5596.4)HTU75:5077.25)HTU70:5372.75)HTU94:5348.55,((WesternFrisian:4447.25,(Latvian:3803.35,(Breton:3182.55,(((Cornish:2343.15,Irish:2118.05)HTU78:1074.45,(AncientIonicGreek:5133.1,ClassicalArmenian:2749.15)HTU80:1612.65)HTU81:1637.85,(Assamese:4333.85,(Faroese:2711.85,Icelandic:6094.35)HTU79:2181.65)HTU82:2715.55)HTU83:1553.55)HTU84:1793.85)HTU85:2041.65)HTU77:5972.7,(((Manx:2196.65,(OldIrish:1582.05,ScottishGaelic:1425.05)HTU51:921.8000000000001)HTU52:3690.4500000000003,(ArmenianWestern:4744.6,(PennsylvaniaDutch:4811.55,((German:3497.65,(LowSaxon:3580.85,(NorthFrisianOomrang:7256.05,OldHighGerman:3579.5)HTU53:2739.15)HTU57:3987.35)HTU59:3091.6,((EnglishAmerican:3250.75,(Norweigan:4312.950000000001,(Gothic:3870.15,Swedish:3067.75)HTU54:1850.65)HTU55:3406.15)HTU58:2090.65,(Avestan:4358.65,Danish:3065.75)HTU60:2416.55)HTU61:2704.25)HTU62:2349.35)HTU63:2720.9500000000003)HTU64:2803.65)HTU56:3340.35,(TocharianA:2586.15,(Samogitian:3216.95,TocharianB:3093.7)HTU68:1428.65)HTU69:4542.65)HTU93:4999.05)HTU95:5333.25)HTU65:7831.65)HTU86:2115.15)HTU87:2193.15)HTU88:3011.75)HTU89:3647.4500000000003)HTU90:2681.75)HTU91:3181.75)HTU92:3932.7)HTU50[156504.3];
