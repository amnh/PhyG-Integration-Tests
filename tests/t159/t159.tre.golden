(Afrikaans:5336.2,(PharasaGreek:4539.35,(Luxemburgish:3080.15,(KolschColognian:4316.0,(GreekModern:6168.549999999999,(ArmenianEastern:3966.05,(AncientGreek:5547.950000000001,(AlbanianTosk:3794.55,(((EnglishBritish:2791.0,(Luwian:4115.5,Yiddish:4234.65)HTU67:2423.35)HTU68:2561.65,((Dutch:1851.15,OldPrussian:1952.35)HTU75:2279.15,((EasternFrisianSaterland:1885.55,Scots:2092.05)HTU71:1774.35,((German:2619.15,(Latgalian:2430.2,Lithuanian:2884.1000000000004)HTU97:1103.55)HTU73:1524.65,(SchwyzerdutschSwissGerman:5546.450000000001,(CentralFranconianRipuarian:3315.15,SouthernTsakonian:4538.55)HTU98:1820.35)HTU74:3308.6)HTU72:1613.65)HTU96:1779.7)HTU76:2010.35)HTU94:2957.15,((WesternFrisian:4302.75,(Latvian:3877.85,(Breton:3158.55,(((Cornish:2358.15,Irish:2117.55)HTU78:1072.95,(AncientIonicGreek:5150.1,ClassicalArmenian:2741.15)HTU80:1619.65)HTU81:1636.85,(Assamese:4331.85,(Faroese:2731.85,Icelandic:6093.85)HTU79:2198.15)HTU82:2692.55)HTU83:1593.05)HTU84:1810.35)HTU85:2205.15)HTU77:4976.7,(((Manx:2580.15,(Samogitian:3447.95,(OldIrish:1541.55,ScottishGaelic:1455.05)HTU51:1190.8000000000002)HTU52:1028.15)HTU53:7288.950000000001,(ArmenianWestern:9304.6,(PennsylvaniaDutch:9526.55,((LowSaxon:3050.35,(NorthFrisianOomrang:7778.55,OldHighGerman:3002.5)HTU54:2047.15)HTU58:10621.8,((SanskritVedic:11744.25,(EnglishAmerican:11291.65,(Norweigan:12936.449999999999,(Gothic:4004.15,Swedish:3191.25)HTU55:10340.15)HTU56:7725.75)HTU59:6940.65)HTU60:6182.15,(Avestan:3688.75,Danish:3840.65)HTU61:9185.55)HTU62:6136.35)HTU63:7126.85)HTU64:5271.45)HTU65:4927.15)HTU57:4978.35,(TocharianB:3199.2,(TocharianA:2132.65,Welsh:2339.85)HTU69:1481.15)HTU70:6269.65)HTU93:4324.05)HTU95:4419.15)HTU66:3516.25)HTU86:2122.15)HTU87:2298.15)HTU88:3076.75)HTU89:3490.4500000000003)HTU90:2689.75)HTU91:3258.75)HTU92:4538.2)HTU50[157437.0];
(Afrikaans:5490.7,(PharasaGreek:4538.35,(Luxemburgish:3074.65,(KolschColognian:4310.5,(GreekModern:6203.049999999999,(ArmenianEastern:4048.55,(AncientGreek:5485.450000000001,(AlbanianTosk:3936.55,((((Luwian:3813.5,(SanskritVedic:3888.35,Yiddish:4041.65)HTU64:1855.15)HTU65:2488.35,(EnglishBritish:3635.1,Gothic:4674.25)HTU66:2096.5499999999997)HTU68:8693.75,((Dutch:1770.25,OldPrussian:1950.25)HTU76:8892.15,((EasternFrisianSaterland:1859.05,Scots:2110.55)HTU73:9074.75,((SchwyzerdutschSwissGerman:13856.85,(CentralFranconianRipuarian:3290.65,SouthernTsakonian:4561.05)HTU98:10051.95)HTU75:7352.8,(Latgalian:2390.2,Lithuanian:3009.6000000000004)HTU97:9431.25)HTU74:6151.65)HTU96:5533.9)HTU77:5333.15)HTU94:5667.65,((WesternFrisian:4478.55,(Breton:3142.45,(((Cornish:2402.15,Irish:2098.05)HTU79:1124.95,(AncientIonicGreek:5154.6,ClassicalArmenian:2800.15)HTU81:1640.15)HTU82:1687.35,(Assamese:4366.85,(Faroese:2735.35,Icelandic:6100.35)HTU80:2174.15)HTU83:2725.05)HTU84:1738.15)HTU85:1843.65)HTU78:5137.55,(((OldIrish:1605.05,ScottishGaelic:1519.05)HTU51:2986.65,(ArmenianWestern:4103.15,(PennsylvaniaDutch:4088.55,((German:3388.65,(LowSaxon:3624.35,(NorthFrisianOomrang:7233.05,OldHighGerman:3624.0)HTU52:2771.65)HTU55:4012.85)HTU57:2393.6,((EnglishAmerican:2505.25,(Norweigan:3755.4500000000003,Swedish:2879.75)HTU53:3234.15)HTU56:1681.65,(Avestan:4222.15,Danish:3169.25)HTU58:1803.55)HTU59:2244.25)HTU60:1993.35)HTU61:2373.4)HTU62:2323.65)HTU54:3043.5499999999997,((Samogitian:2989.35,TocharianB:3296.8)HTU69:1602.15,(Latvian:2309.4,(Welsh:1740.4499999999998,(Manx:1927.25,TocharianA:2630.05)HTU67:1041.15)HTU70:1020.4499999999999)HTU71:1349.15)HTU72:3635.6)HTU93:4535.55)HTU95:4871.8)HTU63:7239.85)HTU86:2300.65)HTU87:2153.15)HTU88:2993.25)HTU89:3530.9500000000003)HTU90:2699.25)HTU91:3266.25)HTU92:4934.7)HTU50[157119.4];
(Afrikaans:4712.2,(PharasaGreek:4556.35,(Luxemburgish:3090.15,(KolschColognian:4328.5,(GreekModern:6217.049999999999,(ArmenianEastern:4014.05,(AncientGreek:5563.450000000001,(AlbanianTosk:3880.55,((((SanskritVedic:3965.15,Yiddish:3917.35)HTU65:1724.15,(EnglishBritish:3547.1,Gothic:4809.75)HTU66:1520.05)HTU67:8524.65,((Dutch:1761.75,OldPrussian:1968.25)HTU73:8743.15,(EasternFrisianSaterland:9765.65,((SchwyzerdutschSwissGerman:13109.85,(SouthernTsakonian:11961.05,(CentralFranconianRipuarian:3632.65,Luwian:4048.1)HTU75:10292.65)HTU98:6723.45)HTU72:7602.3,(Latgalian:2306.2,Lithuanian:3066.1000000000004)HTU97:9009.75)HTU71:5929.65)HTU96:5451.9)HTU74:5425.75)HTU94:5452.15,((WesternFrisian:4719.75,(Latvian:3963.35,(Breton:3395.05,(((Cornish:2436.15,Irish:2006.55)HTU77:1010.9499999999999,(AncientIonicGreek:5093.6,ClassicalArmenian:2802.15)HTU79:1684.65)HTU81:1874.35,((Faroese:2023.85,Icelandic:6635.35)HTU78:1504.15,(Assamese:5086.85,Scots:2364.35)HTU80:1285.65)HTU82:2280.5499999999997)HTU83:1592.55)HTU84:1883.85)HTU85:2082.65)HTU76:5334.2,(((Manx:2180.75,(OldIrish:1577.05,ScottishGaelic:1421.05)HTU51:931.2)HTU52:2995.9500000000003,(ArmenianWestern:4015.2,(PennsylvaniaDutch:4103.55,((German:3406.65,(LowSaxon:3599.35,(NorthFrisianOomrang:7270.55,OldHighGerman:3598.0)HTU53:2748.65)HTU56:4001.85)HTU58:2429.6,((EnglishAmerican:2536.75,(Norweigan:3749.4500000000003,Swedish:2872.75)HTU54:3182.15)HTU57:1709.65,(Avestan:4250.65,Danish:3126.75)HTU59:1825.05)HTU60:2254.75)HTU61:2021.35)HTU62:2398.35)HTU63:2368.65)HTU55:3009.15,((Samogitian:3091.85,TocharianB:3222.8)HTU68:1388.15,(TocharianA:2253.95,Welsh:2233.05)HTU69:1401.15)HTU70:3735.65)HTU93:4730.65)HTU95:5066.65)HTU64:7530.75)HTU86:2257.65)HTU87:2174.65)HTU88:3003.25)HTU89:3537.9500000000003)HTU90:2700.25)HTU91:3265.25)HTU92:4445.7)HTU50[156868.1];
(Afrikaans:4296.2,(PharasaGreek:4558.35,(Luxemburgish:3081.65,(KolschColognian:4323.0,(GreekModern:6199.049999999999,(ArmenianEastern:4006.55,(AncientGreek:5501.950000000001,(AlbanianTosk:3806.55,(((EnglishBritish:3163.1,(Luwian:3799.0,(SanskritVedic:3619.35,Yiddish:4346.65)HTU65:1733.15)HTU66:2536.75)HTU67:9087.65,(Welsh:10341.95,((Dutch:1767.55,OldPrussian:1944.4499999999998)HTU74:9263.65,(EasternFrisianSaterland:10631.85,((German:2196.15,(Latgalian:2553.7,Lithuanian:2904.6000000000004)HTU97:1501.05)HTU72:9913.65,(SchwyzerdutschSwissGerman:13803.449999999999,(CentralFranconianRipuarian:3358.65,SouthernTsakonian:4375.55)HTU98:11136.35)HTU73:8044.6)HTU71:6070.55)HTU96:5925.7)HTU75:5491.25)HTU70:5800.25)HTU94:5523.45,((WesternFrisian:4608.75,(Latvian:3927.85,(Breton:3393.55,(((Cornish:2402.65,Irish:1988.55)HTU77:1012.4499999999999,(AncientIonicGreek:5145.1,ClassicalArmenian:2745.15)HTU79:1635.15)HTU81:1881.85,((Faroese:2013.35,Icelandic:6636.85)HTU78:1511.65,(Assamese:5073.85,Scots:2347.85)HTU80:1283.65)HTU82:2286.0499999999997)HTU83:1600.55)HTU84:1945.35)HTU85:2144.15)HTU76:6157.7,(((Manx:2050.15,(OldIrish:1580.55,ScottishGaelic:1421.55)HTU51:1061.8000000000002)HTU52:3704.9500000000003,(ArmenianWestern:4702.6,(PennsylvaniaDutch:4653.55,((LowSaxon:3204.85,(NorthFrisianOomrang:7594.55,OldHighGerman:3245.0)HTU53:2314.65)HTU57:5290.799999999999,((EnglishAmerican:3582.25,(Norweigan:4246.950000000001,(Gothic:3886.65,Swedish:3039.25)HTU54:1918.15)HTU55:3007.65)HTU58:2241.15,(Avestan:4253.65,Danish:3205.25)HTU59:2803.55)HTU60:2870.75)HTU61:2515.85)HTU62:2844.4500000000003)HTU63:2795.65)HTU56:3379.85,(TocharianA:2624.65,(Samogitian:3223.95,TocharianB:3089.2)HTU68:1397.15)HTU69:4612.65)HTU93:5141.55)HTU95:5404.75)HTU64:7972.15)HTU86:2315.15)HTU87:2235.65)HTU88:3012.75)HTU89:3537.9500000000003)HTU90:2698.75)HTU91:3259.75)HTU92:4186.2)HTU50[156528.2];
(Afrikaans:4370.7,(PharasaGreek:4586.35,(Luxemburgish:2898.65,(KolschColognian:4464.5,(GreekModern:6225.549999999999,(ArmenianEastern:3956.55,(AncientGreek:5589.950000000001,(AlbanianTosk:3839.05,(((EnglishBritish:2689.0,(SanskritVedic:3821.15,Yiddish:4104.35)HTU66:2073.15)HTU67:8675.65,(Welsh:9688.35,((Dutch:1803.15,OldPrussian:1981.35)HTU74:8858.65,((EasternFrisianSaterland:1945.55,Scots:2110.05)HTU71:9148.15,((SchwyzerdutschSwissGerman:13596.35,(SouthernTsakonian:12541.05,(CentralFranconianRipuarian:3663.15,Luwian:4060.6)HTU76:10855.65)HTU98:7071.95)HTU73:8003.8,(Latgalian:2372.2,Lithuanian:2981.6000000000004)HTU97:9637.75)HTU72:6245.75)HTU96:5598.9)HTU75:5078.25)HTU70:5373.75)HTU94:5349.55,((WesternFrisian:4453.75,(Latvian:3807.35,(Breton:3188.55,(((Cornish:2352.15,Irish:2122.05)HTU78:1074.95,(AncientIonicGreek:5139.1,ClassicalArmenian:2753.15)HTU80:1614.15)HTU81:1639.35,(Assamese:4340.85,(Faroese:2717.85,Icelandic:6101.35)HTU79:2182.65)HTU82:2718.05)HTU83:1555.05)HTU84:1795.85)HTU85:2043.65)HTU77:5975.7,(((Manx:2200.65,(OldIrish:1583.05,ScottishGaelic:1426.05)HTU51:924.8000000000001)HTU52:3690.4500000000003,(ArmenianWestern:4749.6,(PennsylvaniaDutch:4816.55,((German:3502.65,(LowSaxon:3586.85,(NorthFrisianOomrang:7260.55,OldHighGerman:3587.5)HTU53:2742.15)HTU57:3989.35)HTU59:3092.6,((EnglishAmerican:3256.25,(Norweigan:4319.950000000001,(Gothic:3875.65,Swedish:3073.25)HTU54:1856.15)HTU55:3411.15)HTU58:2091.65,(Avestan:4367.15,Danish:3069.75)HTU60:2417.55)HTU61:2704.75)HTU62:2349.35)HTU63:2722.9500000000003)HTU64:2805.65)HTU56:3341.35,(TocharianA:2589.15,(Samogitian:3224.95,TocharianB:3099.2)HTU68:1430.15)HTU69:4545.65)HTU93:5001.05)HTU95:5335.25)HTU65:7832.65)HTU86:2115.15)HTU87:2192.65)HTU88:3016.75)HTU89:3659.9500000000003)HTU90:2682.25)HTU91:3188.75)HTU92:4386.7)HTU50[156504.3];
